src/components/diagrams/networkAreaDiagram/network-area-diagram-pane.js

```diff
@@ -21,7 +21,7 @@ export function NetworkAreaDiagramPane({
     disabled,
     visible,
```

Comment:
property 'visible' no more used : to be removed

Reply:
ok, done.